instrumentation/apache-camel-2.20/javaagent-unittests/apache-camel-2.20-javaagent-unittests.gradle

```
@@ -0,0 +1,10 @@
+apply from: "$rootDir/gradle/java.gradle"
+
+dependencies {
+  testImplementation project(':instrumentation:apache-camel-2.20:javaagent')
+  testImplementation group: 'org.apache.camel', name: 'camel-core', version: '2.20.1'
+  testImplementation group: 'org.apache.camel', name: 'camel-aws', version: '2.20.1'
+  testImplementation group: 'io.opentelemetry', name: 'opentelemetry-extension-trace-propagators', version: '0.13.1'
```

You can use the dependency from `dependendcies.gradle` instead:

```suggestion
  testImplementation deps.opentelemetryTraceProps
```